src/main/java/picard/sam/SamToFastq.java

```diff
@@ -133,6 +133,9 @@
             "If the original read is shorter than CLIPPING_MIN_LENGTH then the original read length will be maintained.")
     public int CLIPPING_MIN_LENGTH = 0;
 
+    @Option(shortName = "GZIP", doc = "Compress output FASTQ files using gzip and append a .gz extension to the file names.", optional = false)
+    public Boolean COMPRESS_OUTPUTS = false;

```

Comment:
Consider adding a `mutex` for this argument:
`mutex = {"FASTQ", "SECOND_END_FASTQ", "UNPAIRED_FASTQ"}`